CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ the [releases page](https://github.com/Consensys/teku/releases).
 - Added `--exit-when-no-validator-keys-enabled` command line option.
 - Updated Gnosis chain bootnodes.
 - Support Basic authentication for external signer
+- Added DepositTreeSnapshot download to be used when `--checkpoint-sync-url` option is provided
```

Comment:
- When initialising with the `--checkpoint-sync-url` option, the deposit tree snapshot will also be downloaded, rather than relying on older data packaged in the release.